letsencrypt-auto-source/pieces/bootstrappers/deb_common.sh

```diff
@@ -18,11 +20,18 @@ BootstrapDebCommon() {
   # - Debian 6.0.10 "squeeze" (x64)
 
   $SUDO apt-get update || echo apt-get update hit problems but continuing anyway...
+  
+  # checking if certbot/letsencrypt is already packaged with OS
+  if [ "$QUIET" != 1 ]; then
+      if dpkg --get-selections | grep -v letsencrypt.sh | grep -c letsencrypt -gt 0; then
```

Comment:
Where you currently have the code `grep -c letsencrypt -gt 0`, any of the following would do what you actually want:

```
grep -q letsencrypt                # grep returns true or false based on whether it has any matches`
test `grep -c letsencrypt` -gt 0   # run grep as a subcommand that counts lines, compare the answer to 0
[ `grep -c letsencrypt` -gt 0 ]    # in the unix shell putting something in [ ] is the same as using the "test" command
```
However only the first of these three is really idiomatic bash; I've never seen anyone pipe the output of something into `test` before.